inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/channel/MemoryChannel.java

```diff
@@ -54,12 +54,12 @@ public void push(Message message) {
                 if (message instanceof ProxyMessage) {
                     groupId = ((ProxyMessage) message).getInlongGroupId();
                 }
-                GlobalMetrics.incReadNum(groupId);
+                AgentMetricSingleton.getAgentMetricHandler().globalMetrics.incReadNum(groupId);
```

Comment:
it's better to add a global variate for `AgentMetricSingleton.getAgentMetricHandler().globalMetrics`.

Reply:
Added AgentConstants.GLOBAL_METRICS global field